src/main/twirl/uk/gov/hmrc/play/views/helpers/dropdown.scala.html

```diff
@@ -34,7 +34,7 @@
 }
 <select id="@elements.field.name" name="@elements.field.name" class="@elements.args.get('_selectClass)">
     @if(displayEmptyValue) {
-        <option>@elements.args.get('_emptyValueText)</option>
+        <option @if(elements.args.get('_includeValueAttribute)) {value=""}>@elements.args.get('_emptyValueText)</option>
```

Comment:
@Kerrzinga is there any need to introduce another arg here?

i might be missing something, but would `<option @if(elements.args.get('_emptyValueText)) {value=""}`> do it?

Reply:
as discussed on Slack we'll go with taking the if statement out and making it a breaking change